bin/run_swiftlint

```diff
@@ -2,11 +2,29 @@
 
 echo "--- :swift: Running SwiftLint"
 
-SWIFTLINT_VERSION=$(<.swiftlint.yml awk '/^swiftlint_version: / {print $2}')
-SWIFTLINT_CMD="docker run --rm -v $PWD:/workspace -w /workspace ghcr.io/realm/swiftlint:$SWIFTLINT_VERSION swiftlint"
+SWIFTLINT_ARGUMENTS=(--quiet --reporter relative-path)
+
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --strict | --lenient)
+            SWIFTLINT_ARGUMENTS+=("$1")
+            shift
+            ;;
+        *) break ;;
+    esac
+done
```

Comment:
Indentation consistency: rest of this file uses 2 spaces not 4 🙃

Reply:
🦅 👀 
Fixed on e3a97f68b51d472840e910fb9854fc67e4f38f64.